uber/templates/static_views/legal_name.html

```diff
@@ -14,7 +14,11 @@ <h3 align="center"> Why Does {{ c.EVENT_NAME }} Need My Legal Name? </h3>
 
 <br/> <br/>
 
-Your legal name is for internal use only and will not be printed on public materials (e.g. badges, swag, etc).
+If you do not enter a separate legal name, which is the likely case for most people, we will assume your preferred name is also what will appear on your photo ID.
+
+<br/> <br/>
+
+Both your preferred and legal name are for internal use only and will not be printed on public materials (e.g. badges, swag, etc). We will use your preferred name in emails and when interacting with you, but we still need your legal name to check you in.
```

Comment:
This is actually something we recently found out was NOT entirely true: https://github.com/magfest/ubersystem/issues/2146 This was my bad, as I wrote the original text here without realizing this use-case.

With this PR we can now say that the legal name is not printed on badges, but we can't say the same for preferred name (since that is now printed if you don't fill out a badge_printed_name).